docs/worker/pip-mirror.md

```diff
@@ -0,0 +1,176 @@
+# Installing a Pip Mirror
+
+When running python based lambdas in OpenLambda packages
+are installed with pip. By default, pip installs packages
+through an API to pypi.org, which cannot always be trusted.
+
+Pypi.org cannot guarantee that every indexed package is safe,
+thus a local mirrored index for pip may be beneficial.
+
+### Creating and Defining the Bandersnatch Mirror
+
+In order to setup a bandersnatch pip mirror first install bandersnatch with:
+```
+pip install bandersnatch
+```
+
+Once you have installled bandersnatch, you will have to create the config file.
+To do this run:
+```
+bandersnatch mirror
+```
+
+This will create a bandersnatch.conf file. In this configuration
+file you can set different settings for the mirror.
+
+Furthermore, you can decide the backend for the bandersnatch mirror.
+This can be a docker container, web server (such as nginx), or a filesystem.
+
+OpenLambda only supports a docker container or web server.
+
+For documentation on how to set up the mirror use: [bandersnatch docs](https://bandersnatch.readthedocs.io/en/latest/mirror_configuration.html)
+
+It is important to note that if you do not use an allowlist or 
+blacklist for configuration you will be installing the whole 
+pypi index which is several terabytes.
+
+When deciding which packages to install into the mirror it is important
+to note that the current implementation of the package puller, when
+a pip mirror is defined, is to not use pypi.org even upon install
+failure.
+
+Furthermore, since OpenLambda does not install dependencies with 
+pip's `--no-deps` please ensure that any packages you wish to 
+install have all required dependencies
+within the mirror as well. For more information on how OpenLambda
+installs packages view: [pypi-packages](pypi-packages.md). 
+
+Once you have set up the bandersnatch mirror and are happy with the
+configuration file you can run `bandersnatch mirror` once more to 
+begin installation. This may take several minutes.
+
+### Nginx Support
```

Comment:
Are there shorter directions we can give for nginx?  E.g., is there a one-liner we can use to start a container with nginx?  https://hub.docker.com/_/nginx